projects/packages/my-jetpack/src/products/class-backup.php

```diff
@@ -75,7 +75,11 @@ public static function get_title() {
 	 * @return string
 	 */
 	public static function get_description() {
-		return __( 'Save every change', 'jetpack-my-jetpack' );
+		if ( static::is_active() ) {
```

Comment:
I am still working on fully testing how this behaves - If this `is_active` call depends on the backup plugin being active, I wonder if we would get a scenario where the user has backups active but sees "Your site is not backed up" 🤔

Reply:
I was able to check this, the `Hybrid_Product` class checks if the standalone plugin is active OR if the main Jetpack plugin is active, so this check works fine.